rabbit-escape-engine/src/rabbitescape/engine/BehaviourTools.java

```diff
@@ -182,10 +179,11 @@ public boolean isLeftRiseSlope( Block b )
             || b.type == bridge_up_left;
     }
 
-    private boolean isSolid( Block block )
+    public static boolean isSolid( Block block )
```

Comment:
Since this is now public it might be useful to add some Doxygen to explain what the method does. I've played quite a bit of rabbit escape, but I can't say for sure why anyone cares if a block is "solid" or not (and I can't think what qualifies as a non-solid block).